Client/Frontend/Settings/PasswordDetailViewController.swift

```diff
@@ -129,10 +129,12 @@ extension PasswordDetailViewController: UITableViewDataSource {
             breachDetailView.setup(breach)
             breachDetailView.applyTheme(theme: themeManager.currentTheme)
 
-            breachDetailView.viewBreachDetailsButton.addTarget(self, action: #selector(PasswordDetailViewController.didTapBreachLearnMore), for: .touchUpInside)
+            breachDetailView.onTapLearnMore = { self.didTapBreachLearnMore() }
```

Comment:
To avoid eventual memory leaks, let's avoid capturing a strong reference of self by using `weak self`:
`
breachDetailView.onTapLearnMore = { [weak self] in
                self?.didTapBreachLearnMore()
            }
`